Gemfile

```diff
@@ -70,11 +70,13 @@ gem 'coveralls', require: false
 
 group :development do
   gem 'capistrano-rails', '~> 1.1.1'
-  gem 'pry-byebug'
 end
 
-group :test, :development do
+group :test do
 #	gem "jettywrapper", "~> 1.7"
 #	gem 'rspec-rails', '~> 3.0.0'
+  gem 'pry-byebug'
```

Comment:
Okay. I ran into some weirdness with getting the test suite to pass on Travis based on how those groups were constructed. What gems should we be sharing between the two envs?